### PR TITLE
mockup components of home

### PR DESCRIPTION
**Pull request by solrubado**

### CODE REVIEW

File: frontend/src/components/home/community/Community.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+              <p className={"title"}>Community</p>
+
+              <p className={"text"}>
+                We encourage developers, technologists, cypher punks, coders,
+                crypto-economists, disegners, makers, and the like to apply. Our
+                ecosystem is bottom up. Emergent.
+              </p>
+            </div>
+          </Grid>
+          <Grid item xs={12} sm={6} md={3} lg={3} className={"box-community"}>
```
Comment (ragonzal) [inline frontend/src/components/home/community/Community.js]:
```suggestion
<Grid item xs={12} sm={6} md={3} lg={3} className={"box-community"}>
            <div className={"card-community"}>
              <img src={community} alt="Community" />
              <p className={"title"}>Community</p>

              <p className={"text"}>
                We encourage developers, technologists, cypher punks, coders,
                crypto-economists, disegners, makers, and the like to apply. Our
                ecosystem is bottom up. Emergent.
              </p>
            </div>
          </Grid>
```
Se puede crear un componente stateless y loopear en lugar de escribir todo el componente multiples veces

File: frontend/src/components/home/community/Community.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+              <img src={community} alt="Community" />
+              <p className={"title"}>Community</p>
+
+              <p className={"text"}>
+                We encourage developers, technologists, cypher punks, coders,
+                crypto-economists, disegners, makers, and the like to apply. Our
+                ecosystem is bottom up. Emergent.
+              </p>
+            </div>
+          </Grid>
```
Comment (solrubado) [inline frontend/src/components/home/community/Community.js]:
```suggestion
          <Grid item xs={12} sm={6} md={3} lg={3} className={"box-community"}>
            <div className={"card-community"}>
              <img src={community} alt="Community" />
              <p className={"title"}>Community</p>

              <p className={"text"}>
                We encourage developers, technologists, cypher punks, coders,
                crypto-economists, disegners, makers, and the like to apply. Our
                ecosystem is bottom up. Emergent.
              </p>
            </div>
          </Grid>
```

Comment (solrubado) [reply frontend/src/components/home/community/Community.js]: You can make a stateless component CommunityCard

``
<CommunityCard image={community} title={"Community"} text={textCommunity}/>
``
with CommunityCard being


``
export const CommunityCard = ({image, title, text}) => {
    return (
        <Grid item xs={12} sm={6} md={3} lg={3} className={"box-community"}>
            <div className={"card-community"}>
                Networking
                <p className={"title"}>{title}</p>
                <p className={"text"}>{text}</p>
            </div>
        </Grid>
    )
}
``

**Comment by solrubado** [comment]
- [ ] Refactor code with components
- [ ] Use variables for colors (and text sizes if necessary)
- [ ] Use camelcase for React components

**Pull request closed without merging**